deploy/helm/sumologic/values.yaml

```diff
@@ -153,6 +153,9 @@ sumologic:
       ## Add custom annotations only to setup job pod
       podAnnotations: {}
 
+      ## Time in seconds after which the job result will be removed from the Kubernetes cluster
+      ttlSecondsAfterFinished: 0
```

Comment:
Should this really be 0 by default? We should leave enough time for the user to be able to check the output if the Job failed. I'd put it at at least 60, maybe 120 to be on the safe side.

Reply:
changed